neo/SmartContract/InteropService.NEO.cs

```diff
@@ -73,14 +74,20 @@ private static bool Native_Deploy(ApplicationEngine engine)
             return true;
         }
 
-        private static bool Crypto_CheckSig(ApplicationEngine engine)
+        private static bool Crypto_ECDsaVerify(ApplicationEngine engine)
```

Comment:
Hi Erik, shouldn't we name it "ECDSA" (all caps)?

Reply:
I think `ECDsa` is correct. But I'm not sure.

Reply:
On the internet, I've found 'Ecdsa' and 'ECDSA', but I'm not sure either

Reply:
I think it should be `ECDSA`, because all words represents the method:

Elliptic Curve Digital Signature Algorithm (ECDSA) 
Such as:
Digital Signature Algorithm (DSA)

Can you rename it asap, @lock9?